splunk-to-segment/splunk-to-segment_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSplunkToSegment(t *testing.T) {
+	const (
+		scriptPath   = "../scripts/splunk-to-segment.sh"
+		filePathPass = "sample/fetchujrecordsPass"
+		filePathFail = "sample/fetchujrecordsFail"
+	)
+
+	require.NoError(t, os.Setenv("UID_MAP_FILE", "./sample/getuid"), "Failed to set UID_MAP_FILE")
+	require.NoError(t, os.Setenv("WS_MAP_FILE", "./sample/getworkspace"), "Failed to set WS_MAP_FILE")
+
+	t.Run("PassPath", func(t *testing.T) {
+		assert.True(t, runAndValidateScript(filePathPass, scriptPath), "Script validation failed for PassPath")
```

Comment:
We should have those either as a helper module or a fixture for tests IMO

Reply:
created a seperate testfixture and added test for that fixture as well